dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,8 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
-
+    "puppetlabs.services.master.master-service/master-service": {master-routes: "/puppet"
+                                                                 404:           "/" }
```

Comment:
Minor, but I advise avoiding identifiers that start with a number and/or are entirely numeric.  `[a-z][a-z0-9]*` is usually best, as in the convention for naming a function.  I suggest `http404` or `status404` or `not-found` or something like that as an alternative identifier.